### PR TITLE
[EDU-4110] cli global commands

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue: https://aziontech.atlassian.net/browse/EDU-4110

Changes:

Create a dedicated page for the global commands

### CODE REVIEW

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Azion CLI global options
+description: Azion CLI commands that can be run anywhere in the process.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx:3]:
```suggestion
description: Azion CLI commands that can be run anytime in the process.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+meta_tags: cli, edge, globals, development
+namespace: documentation_cli_globals
+menu_namespace: cliMenuAlpha
+permalink: /documentation/devtools/cli/globals/
+---
+
+Azion CLI commands that can be run anywhere in the process.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx:10]:
```suggestion
Azion CLI commands that can be run anytime during the process.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+---
+
+## version 
+
+```sh 
+azion --version
+```
+
+The `-v` or `--version` option informs which Azion version you've currently installed.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx:20]:
```suggestion
The `-v` or `--version` option informs which Azion version is currently installed.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+The `-c` or `--config` option sets the Azion configuration folder for the current command only, without changing persistent settings.
+
+## log-level 
+
+```sh 
+azion --log-level info
+```
+
+The `-l` or `--log-level` option informs the level of log to be output. The options are:
```
Comment (bru-andrade) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx:44]:
```suggestion
The `-l` or `--log-level` option informs the level of the output logs. The options are: 
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+- `error` 
+- `debug`
+
+## silent 
+
+```sh 
+azion --silent
+```
+
+The `-s` or `--silent` option silences logs completely. Usually, it's used for automation purposes.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx:56]:
```suggestion
The `-s` or `--silent` option silences logs completely. It's often used for automation purposes.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx (hunk 1 above)
Comment (bru-andrade) [reply src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx:3]: Pensei em deixar mais específico

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/globals/globals.mdx [hunk 6]
```diff
@@ -0,0 +1,76 @@
+---
+title: Opções globais da Azion CLI 
+description: Comandos globais da Azion CLI que podem ser utilizados em qualquer etapa do processo.
+meta_tags: cli, edge, globals, development
+namespace: documentation_cli_globals
+menu_namespace: cliMenuAlpha
+permalink: /documentacao/devtools/cli/globals/
+---
+
+Comandos da Azion CLI que podem ser utilizados em qualquer etapa do processo.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/globals/globals.mdx:10]:
```suggestion
Comandos globais da Azion CLI que podem ser executados de qualquer diretório do sistema e entre processos.
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/globals/globals.mdx [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+---
+
+## version 
+
+```sh 
+azion --version
+```
+
+A opção `-v` ou `--version` informa qual versão da Azion você tem instalada.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/globals/globals.mdx:20]:
```suggestion
A opção `-v` ou `--version` informa qual versão da Azion está instalada.
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/globals/globals.mdx [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+A opção `-t` ou `--token` salva um token pessoal fornecido localmente para autorizar comandos do CLI.
+
+## yes 
+
+```sh 
+azion --yes
+```
+
+A opção `-y` ou `--yes` responde automaticamente com sim a todas as interações sim/não.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/globals/globals.mdx:72]:
```suggestion
A opção `-y` ou `--yes` responde automaticamente a todas as interações sim/não com `yes`.
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/globals/globals.mdx [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+A opção `-s` ou `--silent` silencia completamente os logs. Normalmente, é usada para fins de automação.
+
+## token 
+
+```sh 
+azion --token seu-token-pessoal
+```
+
+A opção `-t` ou `--token` salva um token pessoal fornecido localmente para autorizar comandos do CLI.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/globals/globals.mdx:64]:
```suggestion
A opção `-t` ou `--token` salva um personal token fornecido localmente para autorizar comandos do CLI.
```

**Review by bru-andrade — APPROVED** [review]
😎 🤏🏻
😮 👓 🤏🏻

**Review by lfsigreja — APPROVED** [review]
lgtm

**Pull request merged**